codebundles/k8s-namespace-healthcheck/runbook.robot

```diff
@@ -42,6 +42,24 @@ Suite Initialization
     ...    pattern=\w*
     ...    example=(Error|Exception)
     ...    default=(Error|Exception)
+    ${SERVICE_ERROR_PATTERN}=    RW.Core.Import User Variable    SERVICE_ERROR_PATTERN
+    ...    type=string
+    ...    description=The error pattern to use when grep-ing logs for services.
+    ...    pattern=\w*
+    ...    example=(Error: 13|Error: 14)
+    ...    default=(Error:)
+    ${SERVICE_EXCLUDE_PATTERN}=    RW.Core.Import User Variable    SERVICE_EXCLUDE_PATTERN
+    ...    type=string
+    ...    description=Pattern used to exclude entries from log results when searching in service logs.
+    ...    pattern=\w*
+    ...    example=(node_modules|opentelemetry)
+    ...    default=(node_modules|opentelemetry)
+    ${ANOMALY_THRESHOLD}=    RW.Core.Import User Variable    ANOMALY_THRESHOLD
```

Comment:
@jon-funk I was thinking about an x count over y minutes. I think k8s keeps the events for an hour, but a more normal pattern would be something like seeing a pull event 5 times in 5 minutes. Do you think something like that is possible?

Reply:
Example of new rate calc in raised issue:

`Event(s) Per Minute:5.470709500085896 Count:8492 Object:online-boutique/Pod/crashpod-7d4984c5d6-8fzzg Reason:NotTriggerScaleUp Message:pod didn't trigger scale-up: 3 pod has unbound immediate PersistentVolumeClaims`